mix.exs

```diff
@@ -60,23 +60,23 @@ defmodule Membrane.RTC.Engine.MixProject do
   defp deps do
     [
       {:membrane_core, "~> 0.11.2"},
-      {:membrane_webrtc_plugin, "~> 0.12.0"},
+      {:membrane_webrtc_plugin, "~> 0.13.0"},
```

Comment:
Looks like we've got a webrtc / rtp incompatibility here that causes integration tests to fail completely. The same will, I assume, completely break the videoroom.

So, we either:
1. revert this change, or
2. wait for https://github.com/membraneframework/membrane_rtp_plugin/pull/142 to be merged